rustSlint/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # ARGUMENTS --------------------------------------------------------------------
 ##
 # Base container version
-# Using Slint v1.7.2 base images
+# Using Slint v1.8.0 base images
 ##
-ARG CROSS_SDK_BASE_TAG=3.3.1-bookworm-1.7.2
-ARG BASE_VERSION=3.3.1
+ARG CROSS_SDK_BASE_TAG=4.0.0-1.8.0
```

Comment:
Hello @tronical,
I am updating the Slint template to the Torizon OS 7 (which are tag 4), but I am having the following problem on the Slint templates:
```
[+] Running 1/1
 ✔ Container torizon-rustslinttorizon7-1  Created                                                                  0.1s 
Attaching to rustslinttorizon7-1
rustslinttorizon7-1  | Hello Torizon!
rustslinttorizon7-1  | slint linuxkms backend: unrecognized renderer skia, falling back default
rustslinttorizon7-1  | Using Skia OpenGL renderer
rustslinttorizon7-1  | Rendering at 1920x1080
```
Do you know where may be the issue? Should I update the line `ENV SLINT_BACKEND=linuxkms-skia`?

Obs. It shows the application on the screen, however the mouse doesn't work (is it supposed to?)

Reply:
Hello, I tested here and it shows on the screen on arm64 (without the mouse working, not sure why), but on arm32 device (more specifically a colibri-imx6), the application is failing, showing the following message:
```
[+] Running 1/1
 ✔ Container torizon-rustslinttorizon7-1  Created                                                           0.6s 
Attaching to rustslinttorizon7-1
rustslinttorizon7-1  | slint linuxkms backend: unrecognized renderer skia, falling back default
rustslinttorizon7-1  | Hello Torizon!
rustslinttorizon7-1  | Using Skia OpenGL renderer
rustslinttorizon7-1  | Rendering at 800x480
rustslinttorizon7-1  | thread 'main' panicked at /cargo/registry/src/index.crates.io-6f17d22bba15001f/drm-0.12.0/src/control/mod.rs:360:9:
rustslinttorizon7-1  | assertion failed: (has_modifier && modifier.is_some()) || (!has_modifier && modifier.is_none())
rustslinttorizon7-1  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
rustslinttorizon7-1 exited with code 101
```

Reply:
Hello, I removed the line `ENV SLINT_BACKEND=linuxkms-skia` and tested here and it shows on the screen on arm64 (without the mouse working, not sure why), but on arm32 device (more specifically a colibri-imx6), the application is failing, showing the following message:
```
[+] Running 1/1
 ✔ Container torizon-rustslinttorizon7-1  Created                                                           0.5s 
Attaching to rustslinttorizon7-1
rustslinttorizon7-1  | Hello Torizon!
rustslinttorizon7-1  | Using Skia OpenGL renderer
rustslinttorizon7-1  | Rendering at 800x480
rustslinttorizon7-1  | thread 'main' panicked at /cargo/registry/src/index.crates.io-6f17d22bba15001f/drm-0.12.0/src/control/mod.rs:360:9:
rustslinttorizon7-1  | assertion failed: (has_modifier && modifier.is_some()) || (!has_modifier && modifier.is_none())
rustslinttorizon7-1  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
rustslinttorizon7-1 exited with code 101
```
As it was using `linuxkms`, it the `weston` container was not been initiated. Should we use the weston container now?

Reply:
This particular issue happens only
With some GPUs (it's not arm64 vs 43 related). For example I recently saw it on the Mali of the toradex imx95 Evk.

I've got a fix for that and it will be in Slint 1.9.0 in December.

Not sure how to best proceed.

Reply:
Ok, since December is close to now, I think it makes sense to release the Slint templates using Slint 1.9.0, with the fix. Also, this will give us time to do [the change regarding the base Slint containers](https://github.com/torizon/vscode-torizon-templates/issues/271), but moving the `slint-sdk` to the `Dockerfile.sdk` of the container or by moving it to the [torizon containers](https://github.com/torizon/torizon-containers) (we talked with the people responsible for the Torizon containers in the team and they told us that the repo is open for contributions and adding the `slint-sdk` container there is also a possibility).

Reply:
Sounds good. If you want to release an update for Torizon OS 7 before that, I think it would be okay with this being a known issue, as it only affects certain GPUs.

Reply:
On Torizon OS 7 a new flag signalizing that a given template is incomplete/has known issues will be added. So since it only affects certain GPUs we can add it to the release, with this flag set as true. I will open an issue to keep track of the problem

Reply:
Hello @tronical,
Is Slint 1.9 available?

Reply:
We're a few days away from the release :( next week.

Regarding the containers, how can I test torizon 7? (Context: https://github.com/torizon/vscode-torizon-templates/issues/271#issuecomment-2492095517 )

Reply:
Torizon 7 is available both on easy installer and on Torizon Cloud, so just updating the board with the quarterly release should be fine. About the containers, just changing the tags of the base containers to 4, like [here](https://github.com/torizon/vscode-torizon-templates/pull/268/commits/3b9943f6cea0fd2226b85d0b53215068a48112dc), should be fine with it.